packages/lib/modules/pool/actions/remove-liquidity/form/RemoveLiquidityForm.tsx

```diff
@@ -230,6 +235,15 @@ export function RemoveLiquidityForm() {
                   setNeedsToAcceptPIRisk={setNeedsToAcceptHighPI}
                 />
               )}
+              {isV3LBP(pool) && (
```

Comment:
i don't think this is necessary as the price impact for a proportional remove will already be calculated/returned as 0%

Reply:
This is part of the design, only showing the price impact and not the accordion with details.